arch/x86/include/asm/kfence.h

```diff
@@ -12,6 +12,12 @@
 /* TODO: Decide if we want huge page or 4k page on x86. */
 #define KFENCE_POOL_ALIGNMENT PAGE_SIZE
 
+/*
+ * The page fault handler entry function, up to which the stack trace is
+ * truncated in reports.
+ */
+#define KFENCE_SKIP_ARCH_FAULT_HANDLER "asm_exc_page_fault"
```

Comment:
Maybe checking for "page_fault" was a bearable approximation?

Reply:
There are multiple functions with "page_fault" in the name in the stack trace, and therefore doesn't work. Also, for ARM it's going to be different again.